python/test/CMakeLists.txt

```diff
@@ -656,6 +656,7 @@ ot_pyinstallcheck_test (MonteCarloExperiment_std)
 ot_pyinstallcheck_test (CombinatorialGenerator_std)
 ot_pyinstallcheck_test (Experiment_python)
 ot_pyinstallcheck_test (Splitter_std IGNOREOUT)
+ot_pyinstallcheck_test (TensorProductExperiment_std)
```

Comment:
so it seems you're using assert-style tests, you can drop the expout file and use IGNOREOUT here